src/general/onboarding-merchants.md

```diff
@@ -23,7 +23,8 @@
 There are a number of merchant solutions to help vendors take payment in bitcoin and optionally convert to local fiat:
 
 - [Coinos](https://coinos.io/)
-- [Wallet of Satoshi](https://www.walletofsatoshi.com/)
+- [Wallet of Satoshi POS](https://www.walletofsatoshi.com/pos)
```

Comment:
```suggestion
- [Wallet of Satoshi](https://www.walletofsatoshi.com/pos)
```
I think most of these are POS's so don't think we need that in the names.

Reply:
In this case "POS" is part of the app name and I think it is good to diferenciace it to the original WoS app.